Source/WebInspectorUI/UserInterface/Models/Recording.js

```diff
@@ -335,6 +335,11 @@ WI.Recording = class Recording extends WI.Object
         this._displayName = name;
     }
 
+    is2D()
+    {
+        return WI.Recording.is2D(this._type);
+    }
+
```

Comment:
This is clever! I hadn't realized that a `static` method doesn't get overwritten by an identically named method on the object instance. 🤯